nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/UnpackContent.java

```diff
@@ -309,7 +309,10 @@ public void process(final InputStream in) throws IOException {
                             }
                             final File file = new File(tarEntry.getName());
                             final Path filePath = file.toPath();
-                            final String filePathString = filePath.getParent() + "/";
+                            String filePathString = "/";
```

Comment:
You could avoid variable reassignment by moving it to the `else` case of the if statement.